edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss

```diff
@@ -1,7 +1,358 @@
-// Extra rules needed for an LMS theme
+// Extra rules needed for the LMS dark theme
 //
 // The _extras.scss file is imported after all other rules by
 // the LMS Sass. For your own theme, you can override this file
 // to add any custom rules that you need. You can also import
 // partials directly into this file so that you can break your
 // rules into modular pieces.
+
+@import 'certificates';
+@import 'courses';
+@import 'header';
+@import 'navigation';
+@import 'footer';
+@import 'profile';
+@import 'index';
+@import 'dashboard';
+@import 'support';
+@import 'course-experience';
+
+
+//registration
+.view-register .form-actions button[type="submit"]:hover {
+  background-color:#fff;
+  border: 1px solid #007FA3;
+}
+
+
+.register-form {
+  max-width: 600px;
+  margin: auto;
+}
+//generals
+
+body, html, button, textarea {
+  color:$primary-font-color ;
+}
+
+html {
+  background: $inverse !important;
+}
+
+input::not(input[type="button"]) {
+  color:$primary-font-color !important;
+}
+
+select, option, optgroup {
+  color: $primary-font-color !important;
+  background-color: $inverse !important;
+  -webkit-appearance: menulist !important;
+  -moz-appearance: menulist !important;
+  -ms-appearance: menulist !important;
+  -o-appearance: menulist !important;
+}
+
+textarea {
+  color: $primary-font-color !important;
+  background-color: $inverse !important;
+}
+
+a, a:visited:not(.btn) {
+  color: $primary ;
+}
+
+input[type="text"],
+input[type="email"],
+input[type="password"] {
+  color: $primary-font-color;
+}
+
+
+.light-button:hover:not(:disabled),
+div.history-controls input[type="submit"]:hover:not(:disabled),
+a.light-button:hover:not(:disabled),
+div.info-wrapper section.handouts ol li div.hitarea:hover+h4.light-button:not(:disabled),
+div.info-wrapper section.handouts ol li div.hitarea:focus+h4.light-button:not(:disabled),
+.gray-button:hover:not(:disabled),
+input[type="reset"]:hover:not(:disabled),
+input[type="submit"]:hover:not(:disabled),
+input[type="button"]:hover:not(:disabled),
+button:hover:not(:disabled),
+.button:hover:not(:disabled),
+.xmodule_display.xmodule_LTIModule div.lti.wrapper-lti-link
+.lti-link .link_lti_new_window:hover:not(:disabled),
+input[type="submit"].btn-blue:hover:not(:disabled),
+input.btn-blue:hover:not(:disabled),
+button.btn-blue:hover:not(:disabled) {
+  background: $primary;
+  background-image: none;
+}
+
+
+
+.field-label{
+  color: $primary-font-color;
+}
+
+.field-input{
+  background-color: $inverse;
+}
+
+.btn {
+  border-color: $inverse;
+  background: $primary ;
```

Comment:
Can we leave this without the space before the semicolon and keep a space before the value assignment for every property? @luismorenolopera